lib/concurrent/synchronization/lockable_object.rb

```diff
@@ -13,7 +13,7 @@ module Synchronization
                                    when Concurrent.on_rbx?
                                      RbxLockableObject
                                    when Concurrent.on_truffle?
-                                     TruffleLockableObject
+                                     MriMutexLockableObject
```

Comment:
The Synchronisation::Object also has different implementations, it was never designed to be cross used, it may work but it's probably just accidental and may break any time.